File: Baekjoon/step by step/07_2d_array/2566.cpp
// 2566: 최댓값

#include <iostream>

int main() {
  int max_num=0, row_index=0, col_index=0;

  for (int i=0; i<9; i++) {
    for (int j=0; j<9; j++) {
      int input_num;
      std::cin >> input_num;

      if (max_num <= input_num) {
        max_num = input_num;
        row_index = i+1;
        col_index = j+1;
      }
    }
  }

  std::cout << max_num << std::endl << row_index << " " << col_index;

  return 0;
}

File: Baekjoon/step by step/07_2d_array/2566.py
max_num = 0
row_index = 0
col_index = 0

for i in range(9):
  numbers = list(map(int, input().split()))
  for j in range(9):
    if max_num <= numbers[j]:
      max_num = numbers[j]
      row_index = i+1
      col_index = j+1

print(max_num)
print(row_index, col_index)